reference/component-versions.md

```diff
@@ -5,6 +5,82 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.26.0
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision (AMD64) | Revision (ARM64) |
+|----------|--------------|--------------|-------------|
```

Comment:
Since we respinned the charms for 1.26 release last Friday, the revision for each charm is required to update.